near/omni-bridge/src/storage.rs

```diff
@@ -50,15 +50,18 @@ impl TransferMessageStorage {
                 },
                 owner: m.owner,
             },
-            Self::V1(m) => m,
+            TransferMessageStorage::V1(m) => m,
         }
     }
 
     pub fn encode_borsh(
         message: TransferMessage,
         owner: AccountId,
     ) -> Result<Vec<u8>, std::io::Error> {
-        borsh::to_vec(&Self::V1(TransferMessageStorageValue { message, owner }))
+        borsh::to_vec(&TransferMessageStorage::V1(TransferMessageStorageValue {
```

Comment:
Is a clippy change?

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/348/commits/51079c3c136731b4af703f2cfa339a36e80b055b